tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.4.1",
+    "compilerOptions": {
+        "target": "es5",
+        "module": "commonjs",
+        "declaration": false,
+        "noImplicitAny": false,
```

Comment:
also did `npm run build` to get build consistent with yours. 
The constant enum showing different is a language service issue that will be fixed in `1.5`.